core/persistence-api/src/main/java/org/apache/syncope/core/persistence/api/dao/AccessTokenDAO.java

```diff
@@ -34,6 +34,8 @@
 
     AccessToken save(AccessToken accessToken);
 
+    void merge(AccessToken accessToken);
```

Comment:
Change it into
```
AccessToken merge(AccessToken accessToken);
```

Reply:
Shall I keep the method name same(merge) or change it?

Reply:
maybe `replace()` makes more sense